charts/incubator/mixpost/questions.yaml

```diff
@@ -0,0 +1,147 @@
+                                    attrs:
+                                      - variable: APP_NAME
+                                        label: Application Name
+                                        description: The application name
+                                        schema:
+                                          type: string
+                                          required: true
+                                          default: "Mixpost"
+                                      - variable: APP_ENV
+                                        label: Application Name
+                                        description: The application name
+                                        schema:
+                                          type: string
+                                          required: true
+                                          default: "production"
+                                          enum:
+                                            - value: local
+                                              description: Local
+                                            - value: testing
+                                              description: Testing
+                                            - value: production
+                                              description: Production
+                                      - variable: APP_DEBUG
+                                        label: Application Name
+                                        description: The application name
+                                        schema:
+                                          type: string
+                                          required: true
+                                          default: "false"
+                                          enum:
+                                            - value: "true"
+                                              description: "True"
+                                            - value: "false"
+                                              description: "False"
+                                      - variable: APP_URL
+                                        label: Application URL
+                                        description: The application URL
+                                        schema:
+                                          type: string
+                                          required: true
+                                          default: ""
+                                      - variable: APP_KEY
+                                        label: Application Key
+                                        description: The application key generated with https://mixpost.app/encryption-key-generator
+                                        schema:
+                                          type: string
+                                          required: true
+                                          default: ""
```

Comment:
```suggestion
                                      - variable: APP_DEBUG
                                        label: Application Name
                                        description: The application name
                                        schema:
                                          type: boolean
                                          default: false
```

Reply:
APP_KEY needs to be generates in a secrets.tpl
APP_URL can be set automatically.

Reply:
Search `APP_KEY` in the repo if you need examples, basically creates a configmap so the secrets are generated